test/integration/server.h

```diff
@@ -168,6 +177,7 @@ class TestIsolatedStoreImpl : public StoreRoot {
 
   // Stats::StoreRoot
   void addSink(Sink&) override {}
+  void addTagExtractor(const TagExtractor&) override {}
```

Comment:
Wouldn't the list of tag extractors just be a const as well and so we could share that directly?